### ISSUE TITLE
Hair Trigger Horizontal Scrolling in Gmail

### ISSUE THREAD

**Issue by backspaces** [brave/browser-laptop#7643]
- Did you search for similar issues before submitting this one?
Yes

- Describe the issue you encountered:
While using gmail in brave, I get unexpected "scrolling" to the previously viewed email. I tried chrome, and do not seem to have the same problem. 

My current fix is to not use brave for gmail, seems fine for others.

- Platform (Win7, 8, 10? macOS? Linux distro?):
Early 2015 MBP, Sierra.

- Brave Version (revision SHA):
Name	Version
Brave	0.13.5
rev	1db81cb
Muon	2.56.8
libchromiumcontent	56.0.2924.87
V8	5.6.326.50
Node.js	7.4.0
Update Channel	dev
os.platform	darwin
os.release	16.1.0
os.arch	x64

- Steps to reproduce:
    1. Read email w/ window narrow enough for horizontal scrollbar
    2. Scroll within email under ordinary use
    3. It surprisingly pops over to the prior email! 
    
- Actual result:
Unexpected "pop" over to prior email

- Expected result:
Normal scrolling

- Will the steps above reproduce in a fresh profile? If not what other info can be added?
Haven't tried resetting.

- Is this an issue in the currently released version?
Yes

- Can this issue be consistently reproduced?
Yes in the sense that erratic scrolling will occur easily fussing about in gmail.

- Extra QA steps:
    1.
    2.
    3.

- Screenshot if needed:

- Any related issues:

**Comment by backspaces** [comment]
New info: this also occurs in google contacts and google calendar. Does not appear to occur while using chrome rather than brave.

Possibly of use: I note that "pinch zooming" does not work on those pages (mail, calendar, contacts) in brave but does work in chrome. Related?

**Comment by backspaces** [comment]
Hells bells, just happened in twitter! Sounds pretty broad. Have *no* idea why turns up in brave more than chrome. Probably an odd issue with Sierra.

Nowadays I do use brave more than chrome (mainly dev work now) so maybe seeing a problem common to both under the right conditions.

**Comment by Liunkae** [comment]
This was probably a case of horizontal scrolling being interpreted as a two finger back navigate gesture (#7100). This was fixed in 0.14.0.

**Comment by Liunkae** [comment]
This was probably a case of horizontal scrolling being interpreted as a two finger back navigate gesture (#7100). This was fixed in 0.14.0.

Closing as a duplicate of #7100.

**Comment by backspaces** [comment]
Yay! Looks like it fixed it. Thanks 1e6.
